profile/profileedit/index.php

```diff
@@ -1,26 +1,27 @@
 <!DOCTYPE html>
```

Comment:
Same thing here. You should have that `isLoggedIn` check before allowing someone to look at this page